package.json

```diff
@@ -7,51 +7,54 @@
   "engines": {
     "node": ">=4.1.0"
   },
-  "devDependencies": {
+  "dependencies": {
     "banner-footer-webpack-plugin": "git://github.com/dtothefp/banner-footer-webpack-plugin.git",
     "browser-sync": "^2.8.0",
-    "cf-component-demo": "git://github.com/cfpb/cf-component-demo.git#0.8.2",
-    "chai": "^3.3.0",
     "dateformat": "^1.0.11",
     "del": "^2.0.0",
     "gulp": "^3.9.0",
     "gulp-autoprefixer": "^3.0.2",
     "gulp-changed": "^1.2.1",
-    "gulp-concat": "^2.6.0",
-    "gulp-coveralls": "^0.1.4",
     "gulp-cssmin": "^0.1.7",
-    "gulp-eslint": "^1.0.0",
     "gulp-header": "^1.2.2",
-    "gulp-imagemin": "^2.3.0",
-    "gulp-istanbul": "^0.10.0",
     "gulp-less": "^3.0.3",
-    "gulp-load-plugins": "^1.0.0-rc.1",
-    "gulp-mocha": "^2.1.3",
+    "gulp-load-plugins": "^1.1.0",
+    "gulp-imagemin": "^2.3.0",
     "gulp-mq-remove": "0.0.2",
     "gulp-notify": "^2.2.0",
-    "gulp-plumber": "^1.0.1",
-    "gulp-rename": "^1.2.2",
-    "gulp-replace": "^0.5.3",
     "gulp-sitespeedio": "^0.0.8",
```

Comment:
Sadly so, something is still calling for it :( I moved the modules module-by-module and in the current setup this is how it fleshing out. I'd say we get this PR in and then in the remaining modules in the `dependencies` object in `package.json`, flush out why some of the test modules are still required in the production environment.